### DISCUSSION TITLE
cy.intercept not playing nicely with edge functions - started yesterday

### DISCUSSION THREAD

**Discussion by aplchian** [vercel/community#435] [Help] [answered]
Yesterday we started seeing some failed cypress tests, and after looking into it, it seems like the middleware functions don't like what `cy.intercept` is doing. 

I get this when I go to the route `/secure/[blah]` in the cypress test.

![image](https://user-images.githubusercontent.com/7388748/154490521-9d5466ab-c663-4fa4-b4f6-3e2ad3b7952c.png)

cypress intercept:

![image](https://user-images.githubusercontent.com/7388748/154490921-8908e39f-cba0-43ad-9150-428cbe279f49.png)

We just started seeing this yesterday, though, so I'm wondering if a change was made to the edge functions causing this. Any ideas?

**Comment by luisrudge** [comment]
I'm having the same issue. Seems like someone opened an issue about this too: https://github.com/vercel/next.js/issues/34554

**Comment by becca** [comment, marked as answer]
I escalated this to our Eng team this week and believe this should be fixed now. 
Are you still experiencing this issue @aplchian @luisrudge?

**Reply by luisrudge** [reply]
@becca it's fixed for me. thanks!

**Reply by Anag03** [reply]
I'm still having the same issue that was described here [https://github.com/vercel/next.js/issues/34554](url)

**Reply by luisrudge** [reply]
@Anag03 that's true. I'm having this issue as well 🤔

**Reply by becca** [reply]
@Anag03 that link does not work for me. are you referencing this discussion? 
we fixed this issue and i confirmed it was working with @aplchian here. 

If you are experiencing another issue can you please open a bug ticket with clear repro steps and we'll take a peek? 🙇
